src/platform/forms-system/test/js/utilities/ui.unit.spec.js

```diff
@@ -142,9 +142,9 @@ describe('getFocuableElements', () => {
     setOffset('offsetHeight', { configurable: true, value: 10 });
     setOffset('offsetWidth', { configurable: true, value: 10 });
     /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+    /* eslint-disable jsx-a11y/label-has-associated-control */
     const tree = ReactTestUtils.renderIntoDocument(
       <form>
-        <a id="foo">baz</a>
```

Comment:
Figured it would be OK to delete this check rather than override the accessibility rule, since we should not have any links without valid hrefs.